docs/pages/router/features/handoff.mdx

```diff
@@ -0,0 +1,242 @@
+- Handoff requires [universal links](/docs/guides/universal-links) to be configured.
+- Handoff requires the `expo-router/head` component to be used on each page that you want to support.
+
+Ensure your `public/.well-known/apple-app-site-association` file is configured correctly. It must contain the `activitycontinuation` key with the `apps` array containing your app's bundle ID and Team ID formatted as `<APPLE_TEAM_ID>.<IOS_BUNDLE_ID>`, e.g. `QQ57RJ5UTD.app.expo.acme` where `QQ57RJ5UTD` is the Team ID and `app.expo.acme` is the bundle identifier.
+
+```json public/.well-known/apple-app-site-association
+{
+  "applinks": {
+    "details": [
+      {
+        "appIDs": ["<APPLE_TEAM_ID>.<IOS_BUNDLE_ID>"],
+        "components": [
+          {
+            "/": "*",
+            "comment": "Matches all routes"
+          }
+        ]
+      }
+    ]
+  },
+  "activitycontinuation": {
+    "apps": ["<APPLE_TEAM_ID>.<IOS_BUNDLE_ID>"]
+  },
+  "webcredentials": {
+    "apps": ["<APPLE_TEAM_ID>.<IOS_BUNDLE_ID>"]
+  }
+}
+```
+
+The associated domains must be configured and signed, even in development. You can use the `EXPO_TUNNEL_SUBDOMAIN` environment variable to configure the subdomain for your development tunnel. The tunnel is required for testing in development because you need SSL to use universal links, Expo CLI provides built-in support for this by running `npx expo start --tunnel`.
+
+> The environment variable `EXPO_TUNNEL_SUBDOMAIN` must be unique as any user can be using the subdomain at any time. You can use a random string or your GitHub username. However, the URL must match what is in your native `ios/<project>/<project>.entitlements` file under the `com.apple.developer.associated-domains` key.
+
+Ensure you set the Handoff origin in your `app.config.js` file. This is the URL that will be used for the `webpageUrl` when the user switches to your app.
+
+```js app.config.js
+// Be sure to change this to be unique to your project.
+process.env.EXPO_TUNNEL_SUBDOMAIN = "bacon-router-sandbox";
+
+const ngrokUrl = `${process.env.EXPO_TUNNEL_SUBDOMAIN}.ngrok.io`;
+
+/** @type {import('expo/config').ExpoConfig} */
+module.exports = {
+  // ...
+  ios: {
+    bundleIdentifier: "...",
+    associatedDomains: [
+      `applinks:${ngrokUrl}`,
+      `activitycontinuation:${ngrokUrl}`,
+      `webcredentials:${ngrokUrl}`,
+      // Add additional production-URLs here.
+      // `applinks:example.com`,
+      // `activitycontinuation:example.com`,
+      // `webcredentials:example.com`,
+    ],
+  },
+
+  plugins: [
+    [
+      "expo-router",
+      {
+        headOrigin:
+          process.env.NODE_ENV === "development"
+            ? `https://${ngrokUrl}`
+            : "https://my-website-example.com",
+      },
+    ],
+  ],
+};
+```
+
+> Do not use the development-only `?mode=developer` suffix when testing handoff to native.
+
+After you've configured the Expo Config, sync the changes with your native build `npx expo prebuild -p ios`.
```

Comment:
What happens in the case of development build? I assume folks will have to re-create a development build after this step?